chapters/classes.tex

```diff
@@ -409,14 +409,18 @@ \subsection{Conditional Component Declaration}\label{conditional-component-decla
 \cref{interface-compatibility-or-subtyping}).
 
 If the \lstinline!Boolean! expression is false the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
-A component declared with a condition-attribute can only be modified and/or used in connections.
+Such a component can only be modified, used in connections, and/or used in a modifier of another conditional component with a false condition.
+
+There are no restrictions on the component if the \lstinline!Boolean! expression is true.
 
 \begin{nonnormative}
 Adding the component and then removing it ensures that the component is valid.
 
 If a \lstinline!connect!-equation defines the connection of a non-conditional component \lstinline!c1! with a conditional component \lstinline!c2! and \lstinline!c2! is de-activated, then \lstinline!c1! must still be a declared element.
 
 There are annotations to handle the case where the connector should be connected when activated, see \cref{annotations-for-the-graphical-user-interface}.
+
+It is a quality of implementation issue to check whether other values for conditions for conditional could lead to illegal models.
```

Comment:
I cannot quite make sense of this sentence. What are "other values for conditions?" And is "for conditions for conditional" supposed to be "for conditions of a conditional component?"

Reply:
@eshmoylova Can you suggest a suitable rewrite based on what I wrote?

Reply:
Ok, removing then.